components/filecoin/rs/src/message.rs

```diff
@@ -0,0 +1,94 @@
+use fvm_ipld_encoding::RawBytes;
+use fvm_shared::address::Address;
+use fvm_shared::econ::TokenAmount;
+use fvm_shared::MethodNum;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize)]
+#[serde(remote = "fvm_shared::message::Message", rename_all = "PascalCase")]
+pub struct MessageAPI {
+    pub version: i64,
```

Comment:
Just curious, https://github.com/Zondax/filecoin-signing-tools/blob/2e7b005b4733761de11d4a252cb481ca5aedb029/extras/src/message.rs#L16 why isn't version skipped as in https://github.com/Zondax/filecoin-signing-tools/blob/2e7b005b4733761de11d4a252cb481ca5aedb029/extras/src/message.rs#L16?

Reply:
the version is not used for signature calculation, but I added serde(skip) declaration as well to be same, thanks for checking!